### ISSUE TITLE
LED on and off time seems backwards.

### ISSUE THREAD

**Issue by seitmand** [alextaujenis/RBD_Light#10]
I'm using an Arduino UnoR3 in a project and recently added two LEDs.  I'm using your libraries for the 1st time and I can't figure out what I've done wrong. It seems as if the on and off times are reversed.  In the simplified sketch copied below, both LEDs are on most of the time and off for only a short interval when I use statements  which should operate the opposite.  I've included all my #include statements in case some other library is interfering with yours.

Thank you. I greatly appreciate you taking the time to look at this.

```
#include <Wire.h>     //   *** needed ?  **
#include <DHT.h>      // from ladyada
#include <EEPROM.h>
#include <RHReliableDatagram.h> // manager for addressed & confirmed data transmissions
#include <RH_RF95.h>            // driver for LoRa transceiverf
#include <Time.h>   // from Playground?
#include <TimeLib.h>
#include <RBD_Timer.h>
#include <RBD_Light.h>

#define  DIO_PIN           2    // for transceiver
#define  BUZZER_PIN        3    //for piezo crystal
#define  DHT22_PIN         4    // for temperature
#define  LED_RED           5 
#define  LED_GREEN         6 

RBD::Light redLED (LED_RED);
RBD::Light greenLED (LED_GREEN);

void setup() {

//greenLED.blink (150, 1000);  // LED is mostly on
//redLED.blink (150, 3300);  // LED is mostly on
greenLED.blink (1000, 150);  // This LED blinks 3x as often as RED
redLED.blink (3300, 150);   //  Short blink with longer off time

}

void loop() {

  redLED.update();
  greenLED.update();

}
```

**Comment by alextaujenis** [comment]
@seitmand without your full source code it is difficult to identify your issue. You don't need to post the full source... I would suggest you try troubleshooting this.

1. Make sure the [RBD::Light Example Sketch](http://robotsbigdata.com/docs-arduino-light.html#example-wiring) can run on your Arduino and circuit (without any other code). 
1. Open your existing project and comment-out all of the code that doesn't run the lights and remove all of the other libraries. Then, put the first library back in and uncomment the Wire.h code. Test the lights. Keep doing this until you find the library or code that is interfering with the RBD::Light blink.

Just a side note: RBD::Light will not function correctly if any of the libraries or your code has delay().

**Comment by seitmand** [comment]
@alextaujenis.  Thank you for your great suggestions.  I just pulled my entire assembly apart so I could better troubleshoot the hardware (as I was fairly sure my software was written correctly).  And, lo and behold, I had wired the LEDs backwards putting them into the 5volt line instead of the GROUND.  And, I had also reversed the LED polarities, which is why they still lit.

Rather than adding a line into the library to allow for this backwards construction, I decided to unsolder all the related components and put them together the way I originally planned.  (Since I'm hoping to produce a few of these devices, I thought it would be best to make them all the same.)  Now everything works as hoped for.

I had not gotten to updating my query yet and your response beat me to it.  Thanks again.
